lib/streamlit/web/server/routes.py

```diff
@@ -160,6 +160,41 @@ async def get(self):
             self.write(msg)
 
 
+# NOTE: We eventually want to get rid of this hard-coded list entirely as we don't want
+# to have links to Community Cloud live in the open source library in a way that affects
+# functionality (links advertising Community Cloud are probably okay 🙂). In the long
+# run, this list will most likely be replaced by a config option allowing us to more
+# granularly control what domains a Streamlit app should accept cross-origin iframe
+# messages from.
+ALLOWED_MESSAGE_ORIGINS = [
+    "https://devel.streamlit.test",
+    "https://share.streamlit.io",
+    "https://share-demo.streamlit.io",
+    "https://share-head.streamlit.io",
+    "https://share-staging.streamlit.io",
+    "https://*.demo.streamlit.run",
+    "https://*.head.streamlit.run",
+    "https://*.staging.streamlit.run",
+    "https://*.streamlitapp.test",
+    "https://*.streamlitapp.com",
+    "https://*.streamlit.run",
+    "https://*.demo.streamlit.app",
+    "https://*.head.streamlit.app",
+    "https://*.staging.streamlit.app",
+    "https://*.streamlit.app",
```

Comment:
Note that these domains were added as we'll soon be hosting apps in Community Cloud on `streamlit.app` over `streamlitapp.com`

Reply:
When is that targeted? Im weary of a window where someone is able to claim these domains and pwn us until we cut over . Can we add these when needed ?

Reply:
Fairly certain it's targeted for very soon + we've already registered the domain so there shouldn't be any risk here, but looping in @sfc-gh-atoader to confirm

Reply:
Confirmed. We can release as soon as Monday, but we'll wait for this to go out, upgrade all apps to use the latest version of Streamlit, and then upgrade